astro/src/content/docs/_shared/_release-notification.astro

```diff
@@ -1,68 +1,58 @@
-<script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/634739.js"></script>
-
-<script type="module">
-document.addEventListener('DOMContentLoaded', () => {
-    const form = document.getElementById('myActualForm');
-    const hsContextInput = document.getElementById('hs_context');
-
-    if (!form || !hsContextInput) return;
-
-    function getCookie(name) {
-        const match = document.cookie.match(new RegExp('(^| )' + name + '=([^;]+)'));
-        return match ? match[2] : null;
-        }
-
-    function setHubSpotContext() {
-        const hutk = getCookie('hubspotutk');
-        if (!hutk) {
-            console.warn('HubSpot tracking cookie not found.');
-            return;
-            }
-
-        const hsContext = {
-            hutk,
-            pageUrl: window.location.href,
-            pageName: document.title
-            };
-
-        hsContextInput.value = JSON.stringify(hsContext);
-        }
-
-    form.addEventListener('submit', () => {
-        setHubSpotContext();
-
-        const formContents = form.querySelector('div.flex.gap-x-4');
-        if (formContents) {
-            formContents.style.display = 'none';
-            }
-
-        if (!form.querySelector('.thank-you-message')) {
-            const thankYouMessage = document.createElement('p');
-            thankYouMessage.textContent = 'Thank you for your submission!';
-            thankYouMessage.className = 'thank-you-message text-white mt-4';
-            form.appendChild(thankYouMessage);
-            }
-        });
-    });
+<script>
+  import HubSpotForm from "src/components/HubspotForm";
+  new HubSpotForm("#release-notification","c8db8f9b-f7f1-40d6-ae85-88fba80822bc");
 </script>
 
 <p>There are a number of ways to be notified of new releases:</p>
 <ul>
-<li>The <a href="/docs/release-notes">release notes</a> are updated when there is a new release or shortly thereafter.</li>
-<li>There is an <a href="/docs/releases.xml">RSS feed of releases</a> to which you can subscribe.</li>
-<li>The <a href="/blog/tag/release-notes/">blog has release announcement posts</a> and you can subscribe to an <a href="/blog/feed.xml">RSS feed</a>.</li>
-<li>You can subscribe to our release announcement email list using the form below.</li>
+  <li>
```

Comment:
Why did you break these into new lines? Seems like the same content, would be nice to not have it show up as a change.